Formula/f/fava.rb

```diff
@@ -165,8 +166,8 @@ class Fava < Formula
   end
 
   resource "pdfminer2" do
-    url "https://files.pythonhosted.org/packages/55/18/0a856200ff54db80612f66878f9b037ddb9c873f885bc58339b528611994/pdfminer2-20151206.tar.gz"
-    sha256 "7d05aa3dd1e779080fef13aef454501b51a3f7649d7f18e78c640bdbd34e1e77"
+    url "https://files.pythonhosted.org/packages/e0/55/5e235321d7494772264b577a8569c102b9d9ef867f7239d14d562e89bed9/pdfminer2-20151206.macosx-10.10-x86_64.tar.gz"
+    sha256 "9c0599bfde105a8d58e3f679c31ab84871dc9bd3debf1be6511e1abaa4db867f"
   end
```

Comment:
This bit needs to be reverted. Don't know why this happens.

Reply:
yeah, not quite sure why it picks this artifact over the other cc @Homebrew/core

Reply:
That's `pdfminer2`'s fault of uploading binary as sdist - https://pypi.org/project/pdfminer2/#files

The package itself is no longer maintained so probably will never be fixed.

However, the usage here should go away in future: https://github.com/beancount/beancount/commit/dc761f986c32bbb109cb719969ae55d6eb7247ae